### ISSUE TITLE
[Expressive theme] Review Percy diff from feature branch

### ISSUE THREAD

**Issue by shixiedesign** [carbon-design-system/carbon-for-ibm-dotcom#6125]
When the Feature branch from #6121 is available and expressive theme import removed #6122, we should see a lot of diffs between the current build and feature branch build. Designers and dev should both 

1. Review the diff to identify:
2. What to fix, & create issues against individual component (design spec fix, dev code fix)

**Comment by oliviaflory** [comment]
I have reviewed most of the components that were altered from the removal of the expressive theme and opened dev issues for things that need to be updated. 

We reviewed a few components with the design team on Thursday May 27th that needed team input, and need to review Locale selector before closing:

- [ ] Footer the removal of expressive theme made button, combo box & accordion text smaller. Do we open a bug to fix (July release) or wait for Dragos' Footer v2 design updates (date?). [Olivia] Decision: Footer v2 will update designs
- [ ] Leaving IBM removal of expressive theme. Update label and link, but ok to leave modal button? 6205 [Olivia]
- [ ] Tabs extended overriding Carbon accordion 6206 [Olivia]
- [ ] Feature card Featured section – eyebrow size [Olivia]  
- [ ] Locale selector Search field, OK to be 14? Should we reduce list items below? [Olivia]
